### ISSUE TITLE
Feature Request: Add Alias for Groups

### ISSUE THREAD

**Issue by e-scheer** [hoophq/hoop#526]
When groups are retrieved from an IDP like Azure, they are provided as group IDs, which are not user-friendly. This makes it challenging to identify and manage groups within the application.

Would it be possible or is it planned to add an alias option to rename these groups? This would allow for a more user-friendly naming convention without changing the underlying group ID from the IDP.

This feature would be particularly helpful for integrations where the group IDs are not descriptive or intuitive for end users.

Thanks !

**Comment by sandromello** [comment]
Hey @e-scheer , it's possible to propagate groups as names with Azure. Have you taken a look in [this documentation](https://hoop.dev/docs/configure/idp/azure#propagating-groups-as-names)?

**Comment by e-scheer** [comment]
Yes, I've tried that, but unfortunately, the solution only works with ApplicationGroup, which is limited to users with an AD subscription.

By the way, kudos for the documentation! It's concise and detailed—something I haven't seen in a while. Thank you for that.

**Comment by eschdeuse** [comment]
Hello,

Just coming back to this, has it been discussed internally ? For example a simple text field here (optional, by default empty) that if set will take precedence to the group visual name.

![Image](https://github.com/user-attachments/assets/766a3b1c-319c-4e06-9d54-662fbc13c2c1)
